Shared/Blazorise.Shared/Models/Employee.cs

```diff
@@ -28,24 +28,41 @@ public Employee( Employee other )
         Zip = other.Zip;
     }
 
+    [Display( Name = "Id" )]
     public int Id { get; set; }
 
     [Required]
+    [Display( Name = "First Name" )]
```

Comment:
What do you think we generate `Caption` from the field name if `DisplayAttribute` is not defined?

Example:

- `FirstName` to `"First Name"`
- `DateOfBirth` to `"DateOfBirth"`
- `City` to `"City"`

Reply:
We take in the property name already. But I'm guessing you gave the example for the Date Of Birth incorrectly?
It should be `DateOfBirth` to "`DateOfBirth`" 
Was that what you mean? Well, us trying to guess will only work correctly if people use Properties with PascalCase... 
Then what if they don't want our smartass generation? Then they are stuck with names they don't want.